tests/components/bottom_bar.test.ts

```diff
@@ -66,7 +66,7 @@ describe("BottomBar component", () => {
     const dispatch = jest.spyOn(model, "dispatch");
     mockUuidV4To(model, 42);
     const activeSheetId = model.getters.getActiveSheetId();
-    triggerMouseEvent(".o-add-sheet", "click");
+    await click(fixture, ".o-add-sheet");
```

Comment:
This will slow down the tests no ? We don't need to wait for a new render. (same for all the other `triggerMouseEvents` not followed by nextTick.